requirements.txt

```diff
@@ -8,4 +8,6 @@ bottle
 requests
 keras-tuner
 # below, to support birdnet_analyzer.gui
-matplotlib===3.9.3
\ No newline at end of file
+matplotlib===3.9.3
+# below, for running tests when developing birdnet-analyzer itself
+pytest==8.3.4
```

Comment:
I originally added this as `requirements-dev.txt`, but then thought to keep-it-simple by leaving it here